demo/src/app/components/+typeahead/typeahead-section.list.ts

```diff
@@ -77,6 +78,17 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/dropup/dropup.html'),
         outlet: DemoTypeaheadDropupComponent
       },
+      {
+        title: 'No result',
+        anchor: 'no-result',
+        description: `
+         <p>Fired on every key event and returns true in case of matches are not detected. To see message
+         "No Results Found" enter the value of which is not in the list</p>
```

Comment:
enter the value of which is not in the list - > enter the value that doesn't match anything from the list